bin/bootstrap

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+carthage bootstrap --platform iOS --no-use-binaries
```

Comment:
AFAIK, we've been doing it this way for a _really_ long time now. All of my most recent projects have used the binaries created by Carthage. Is there something I don't know?